src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,43 @@
+.nav {
+  box-sizing: border-box;
+
+  &__link {
+    text-decoration: none;
+    color: $nav-link-color;
```

Comment:
The variable `$nav-link-color` is used here, but it is not defined within this file. Ensure that this variable is defined in a global SCSS file or imported correctly to avoid compilation errors.